src/components/Card.vue

```diff
@@ -126,3 +259,22 @@ p.bold {
   border-top: solid #8300bf;
 }
 </style>
+
+<style>
```

Comment:
Why is this a separate style and not scoped?

Reply:
This is for the popper, popper in element ui is at the body level of the html (unless specified otherwise) so anything scoped will not be used there as the html element itself will not have the data-* tag. 
I decided to have this in the body level so the tooltip can extend beyond the area of this component.